package.json

```diff
@@ -36,6 +36,10 @@
     "pathe": "^1.1.0",
     "unbuild": "^1.2.1"
   },
+  "peerDependencies": {
+    "@nuxt/kit": "^3.5.0",
+    "nuxi": "^3.5.0"
```

Comment:
Considering nuxi is an implicit dependency of nuxt this would require a change in template and existing users if released as semver minor — but I guess is right thing to do eventually requiring direct nuxi dep